public/styles/styles.css

```diff
@@ -0,0 +1,109 @@
+body {
+  background-image: url(https://s-media-cache-ak0.pinimg.com/originals/25/62/a3/2562a3c4587e6ee34783bd1fa820ace9.jpg);
+  background-attachment: fixed;
+  font-family: 'Special Elite', cursive!important;
+  color: #000;
+
+}
+
+
+h1 {
+  font-family: 'Special Elite', cursive!important;
```

Comment:
!important is something you mostly shouldn't have to use, because you shouldn't have conflicting styles. The fact that it's on so many of your declarations scares me.